example/credentials.example.yaml

```diff
@@ -1,3 +1,4 @@
 app-id: <Your app ID>
 app-key: <Your app key>
-app-secret: <Your app secret>
\ No newline at end of file
+app-secret: <Your app secret>
+app-cluster: <Your app cluster. Omit for default>
```

Comment:
I'm under the impression several other libraries assume a default cluster which is why I think I had it set up like this. That said, I've been bitten by the dash setting me up to an EU cluster, clients assuming a US cluster and error messages being slightly unhelpful before, so I wouldn't mind erring on the side of explicitness.